zerver/views/development/registration.py

```diff
@@ -87,8 +87,9 @@ def register_development_realm(request: HttpRequest) -> HttpResponse:
 @csrf_exempt
 def register_demo_development_realm(request: HttpRequest) -> HttpResponse:
     count = UserProfile.objects.count()
-    name = f"user-{count}"
-    email = f"{name}@zulip.com"
+    name = f"demo-admin-{count}"
```

Comment:
I felt like it would help to differentiate these users from ones created in the normal new realm creation flow.

Reply:
Sure, that seems very reasonable.